CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ endif()
 
 project(openvino_tokenizers)
 
+option(CPACK_INCLUDE_TOPLEVEL_DIRECTORY "Boolean toggle to include/exclude top level directory." OFF)
+if(WIN32)
+    set(CPACK_GENERATOR "ZIP" CACHE STRING "List of CPack generators to use.")
```

Comment:
let's also change this default in OpenVINO

Then, in `master` we can drop this setting later

Reply:
See https://github.com/openvinotoolkit/openvino/pull/25042